docker-compose.yml

```diff
@@ -48,9 +48,6 @@ services:
       - db
       - redis-store
       - redis-cache
-      - statsd
-      - localstack-s3
-      - frontend
     command: web-dev
```

Comment:
Instead of doing this, can you create a `test` service like this?:

```
test:
  extends:
    service: base
  ports:
    - "8000:8000"
  links:
    # Only add services we need to run the tests.
    - db
    - redis-store
    - redis-cache
```

Then you can change `make run` back to what it originally was and change the test script to run in the `test` container:

```
docker-compose run -e DEVELOPMENT -e DJANGO_CONFIGURATION $CI_ENV test test $@
```